### ISSUE TITLE
Responsive break in https://docs.djangoproject.com/en/1.10/intro/tutorial05/

### ISSUE THREAD

**Issue by pckrishnadas88** [django/djangoproject.com#729]
The design breaks in mobile view for the below page. The bottom language & documentation version causes horizontal scroll in this page. I used Google chrome on my Redmi note 3 device.

![screenshot_2016-12-22-21-20-17-084_com android chrome](https://cloud.githubusercontent.com/assets/411869/21433348/88c097fe-c83d-11e6-9c4e-af2c097ad632.png)

You can see the pagelink here : ​https://docs.djangoproject.com/en/1.10/intro/tutorial05/

**Comment by timgraham** [comment]
Is this the only page in the documentation where you see the behavior? Just loading the page and it looks like this? I cannot reproduce the issue on my device.

**Comment by pckrishnadas88** [comment]
Yes only in this page.  You can see this from desktop chrome browser. Click on inspect element and select mobile devices. See the screenshot from the desktop google chrome.

https://drive.google.com/file/d/0B6aIJfXwADFjV1QwdVZmb2w5aUk/view?usp=sharing

**Comment by pckrishnadas88** [comment]
Found out what was causing the issue. The text was overflowing at two places which are shown in the screenshot. I tried deleting those sections then the issue is fixed. Please have a look at these images.
![overflow--section2](https://cloud.githubusercontent.com/assets/5269476/21434776/4004d3d6-c89c-11e6-82ef-61bb3568164d.png)
![overflow-section1](https://cloud.githubusercontent.com/assets/5269476/21434777/4005941a-c89c-11e6-82d3-106e48fd4b9f.png)

**Comment by raphamendonca** [comment]
Hi,

I found the same problema on the Download page.

![screenshot from 2017-04-14 10-09-00](https://cloud.githubusercontent.com/assets/807571/25044017/7d33d7fa-20fa-11e7-90b1-c99b977d53e3.png)

Appears to be a fixed width in a tag < p > just remove this could work.

**Comment by knyghty** [comment]
This also happens on some docs pages, e.g. https://docs.djangoproject.com/en/3.0/ref/templates/api/

It's caused here by the long names in the sidebar, e.g. `django.contrib.messages.context_processors.messages`.

It can be fixed by setting `overflow-wrap: break-word` somewhere relevant, but this fix isn't beautiful as it doesn't break in very good places. I'm not sure if it's possible to add `&shy;` in those strings somewhere.

<img width="491" alt="Screenshot 2020-05-23 at 14 30 29" src="https://user-images.githubusercontent.com/3871354/82730698-06ee5700-9d02-11ea-8cc1-f50e7ff1c25d.png">

**Comment by knyghty** [comment]
This is still an issue that should be fixed.

**Comment by knyghty** [comment]
Sh
